entrypoint.py

```diff
@@ -42,12 +42,14 @@
     elif poetry == "yes" and poetry_version != "":
         pip_install_poetry = subprocess.run(["pip", "install", f"poetry=={poetry_version}"], capture_output=True)
         poetry_config = subprocess.run(["poetry", "config", "virtualenvs.create", "false"], capture_output=True)
-        poetry_install = subprocess.run(["poetry", "install", "--no-dev", "--no-root"], capture_output=True)
-        if pip_install_poetry.returncode != 0 or poetry_config.returncode != 0 or poetry_install.returncode != 0:
+        poetry_install = subprocess.run(["poetry", "install", "--only", "main", "--no-root"], capture_output=True)
+        poetry_requirements = subprocess.run(["poetry", "export", "-f", "requirements.txt", "-o", "requirements.txt"], capture_output=True)
```

Comment:
why is it needed to export the requirements.txt?

Reply:
will we need similar pr's for the different python versions (currently in different branchs & require seperate releases)?

maybe we should have a script to update them?

Reply:
Seems to be that in some part of the Goblet code, "requirements.txt" is required. At least when I was trying to deploy I got this message:

`Missing requirements.txt. Make sure you are in the correct directory and this file exists`

I supposed that this requirements.txt is the one that is required by the code being deployed.

Reply:
> will we need similar pr's for the different python versions (currently in different branchs & require seperate releases)?
> 
> maybe we should have a script to update them?

Oh sure, by now I can do it manually, while I figure out a more abstract and reutilizable way of doing a script that makes port. Anwyay I know @mauriciowittenberg is working on make a more elegant way of deal with the GHA and python versions stuffs

Reply:
sounds good. requirements.txt is required for cloudfunctions so maybe that why. 

poetry which is normally used by cloudrun shouldnt need it, but it shouldnt either way in adding this.